deepspeed/runtime/engine.py

```diff
@@ -114,12 +112,11 @@
 
 
 def split_half_float_double_sparse(tensors):
-    device_type = get_accelerator().device_name()
     supported_types = [
-        "torch.{}.HalfTensor".format(device_type),
-        "torch.{}.FloatTensor".format(device_type),
-        "torch.{}.DoubleTensor".format(device_type),
-        "torch.{}.BFloat16Tensor".format(device_type),
+        "torch.cuda.HalfTensor",
```

Comment:
This is undoing the accelerator abstraction PR. Maybe this is a bad conflict resolution?